l1-contracts/test/portals/TokenPortal.sol

```diff
@@ -55,9 +55,49 @@ contract TokenPortal {
     return inbox.sendL2Message{value: msg.value}(actor, _deadline, contentHash, _secretHash);
   }
 
+  /**
+   * @notice Deposit funds into the portal and adds an L2 message which can only be consumed privately on Aztec
+   * @param _amount - The amount to deposit
+   * @param _deadline - The timestamp after which the entry can be cancelled
+   * @param _secretHashForL2MessageConsumption - The hash of the secret consumable L1 to L2 message. The hash should be 254 bits (so it can fit in a Field element)
+   * @param _secretHashForL2MessageConsumption - The hash of the secret to redeem minted notes privately on Aztec. The hash should be 254 bits (so it can fit in a Field element)
+   * @param _canceller - The address that can cancel the L1 to L2 message.
+   * @return The key of the entry in the Inbox
+   */
+  function depositToAztecPrivate(
+    uint256 _amount,
+    uint32 _deadline,
+    bytes32 _secretHashForL2MessageConsumption,
+    bytes32 _secretHashForRedeemingMintedNotes,
+    address _canceller
+  ) external payable returns (bytes32) {
+    // Preamble
+    // @todo: (issue #624) handle different versions
```

Comment:
You can get rid of these, think they are stale, or at least not relevant while we don't know the exact upgrade mechanism if any.